west.yml

```diff
@@ -92,7 +92,7 @@ manifest:
       revision: 1080094bc246357c6c35e80e8b2d5d0ce7e0d963
       path: tools/net-tools
     - name: hal_nxp
-      revision: 60fe8c0ae6e24d165cf42770b95c7df22b778386
+      revision: pull/39/head
```

Comment:
Adding DNM label until the hal pr gets merged and this gets updated to the new SHA.

Reply:
Yep, once zephyrproject-rtos/hal_nxp#39 is merged, I'll change the SHA accordingly.

Reply:
It's been merged now

Reply:
> It's been merged now

Thanks @MaureenHelm , I've seen it and prepared the west.yml update. I'd need an input from @galak  , so I don't do two pushes, one with the west.yml changes and one with the potential DT flag changes